src/web/routes/entities/schemas.ts

```diff
@@ -22,7 +21,16 @@ export const ASSOCIATE_ENTITY_REQUEST_SCHEMA: JSONSchemaTypeWithId<{
 					},
 					required: ['url'],
 				},
-				associateWith: ATLASSIAN_ENTITY_SCHEMA,
+				associateWith: {
+					type: 'object',
+					properties: {
+						ati: { type: 'string', pattern: 'ati:cloud:jira:issue' }, // Handle only associations with a Jira Issue.
```

Comment:
**note:** Return `HTTP 400 Bad Request` if `ati` is not `ati:cloud:jira:issue` on the entry point to the app. Therefore, the remaining code does not need to worry about the type of entity a Design is associated to.